src/Adherent.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+
+namespace Ipssi\Evaluation;
+use DateInterval;
+
+
+class Adherent
+{
+    private $nom;
+    private $pret;
+
+    public function __construct(string $nom, array $pret = array())
+    {
+        $this->nom = $nom;
+        $this->pret = $pret;
+    }
+
+    public function emprunter(Bibliotheque $bibliotheque, string $souhait, \DateTime $date = null)
```

Comment:
```suggestion
    public function emprunter(Bibliotheque $bibliotheque, string $souhait, \DateTime $date = null): Pret
```
La méthode pourrait retourner un objet Pret ou throw des esception pour les cas impossibles